### ISSUE TITLE
Heavy crossbow could use damage tweaking

### ISSUE THREAD

**Issue by FayFlightee** [CleverRaven/Cataclysm-DDA#53995]
### Is your feature request related to a problem? Please describe.

Heavy crossbow, which has so strong tension they need a pulley to even pull the string back, currently deals the same damage as a composite crossbow that can be hand-drawn with not so crazy 9 strength *5* times faster than reloading a heavy crossbow.

### Solution you would like.

The heavy crossbow could use more damage. Maybe 10, at the very least. Sure it might be close to compound crossbow's 13, but heavy crossbow still takes 3 times longer to reload and is quite a bit more bulky and heavier.

**Comment by Drew4484** [comment]
Find a particular example of a crossbow that you think is suitable, if you can determine the energy output that can be used to model the appropriate damage.

**Comment by FayFlightee** [comment]
I'm not a bow expert but I'll see what I can find.

**Comment by FayFlightee** [comment]
Okay I have no idea about irl value (but if someone can find out, that'd be appreciated), but at least it could use an extra point. Wood crossbow have 8 str requirement and do 4, composite have 9 and it does 6 (base damage). At least heavy crossbow can do a tiny bit better than exactly the same as composite bow.

**Comment by estebandellasilva** [comment]
https://www.youtube.com/watch?v=bU2SkGz0RYM

Dunno if this is exactly what is needed

**Comment by Drew4484** [comment]
Any change that doesn't have some research behind it is not likely to get approved. Take a look at #37241 as an example of archery rebalancing. This isn't to discourage you, just show you that changes aren't usually made on "x feels weak".

**Comment by Testudo-Cultu** [comment]
There's some information on both the composite and heavy crossbow in data/json/items/ranged/crossbows.json. Unfortunately, real-world data is hard to find with these specifications.

"800 lbs, 40in steel limbs, 0.41 Slugs with a 30g bolt, 2.0 slugs with a special 0.5 lb bolt" - Heavy crossbow

Note that this doesn't include the powerstroke. Using data from Tod's Workshop and historical data of heavy crossbows and metal limbed crossbows, the powerstroke is probably about 7 inches.

"160lb draw, 16in powerstroke, 34in limbs, 51J, 0.394 slugs with 30 gram bolt" - Composite crossbow

The composite crossbow says its energy is 51 joules, or at least I assume. I don't know what else "51J" means otherwise. Comparing this to a modern 160 lbs draw weight composite crossbow's ~61 joules of kinetic energy means that this is probably at least close. With data from Tod's Workshop on a 1,000 lbs crossbow (200 lbs over the in-game crossbow, so that's a discrepancy), a 30 gram bolt would only have about 35.6 joules of energy.

The weight of an individual wooden bolt in-game is 25 grams, which is pretty standard for a modern crossbow bolt, but not correct for a heavy crossbow like this. Based on Tod's Workshop's data, the kinetic energy of a 1,000 lbs draw weight crossbow should be more like 100 joules.

So, in other words, the heavy crossbow is actually garbage if it uses 25-30 gram bolts, and should have even worse damage. However, if it used the correct ~70-90 gram bolts, it would in theory have about twice the kinetic energy of the composite crossbow.

**Comment by FayFlightee** [comment]
So to put it correctly, heavy crossbow is better with actual heavier bolts for its size... which the said heavy bolts isn't a thing in the game.

**Comment by Testudo-Cultu** [comment]
> So that means heavy crossbow is better with actual heavier bolts for its size... which the said heavy bolts isn't a thing in the game.

Yeah, in terms of per-bolt kinetic energy. It would probably still be lower in DPS because of the longer reload time, but I think that gives both crossbows a purpose. 

This isn't a hard thing to add. All it means is that heavy crossbows should have their own ammo type. Heavy bolts would be basically the same as regular bolts, but much heavier and larger. Sort of like 7.62x51mm bullets are to 5.56x45mm.

**Comment by FayFlightee** [comment]
Let's be honest when you're reloading your crossbow you run away quite a lot anyways, I figure DPS isnt much of a thing with crossbows for me. Alright, new bolts for it it is.

**Comment by FayFlightee** [comment]
Let's be honest when you're reloading your crossbow you run away quite a lot anyways, I figure DPS isnt much of a thing with crossbows for me. Alright, new bolts for it it is.

**Comment by estebandellasilva** [comment]
Dunno if this was the idea or not but - should heavy crossbow bolts have higher armor penetration than their standard/compound brothers? Because from the video he says that they have higher delivering power - with a heavier bullet this would equate to higher penetration

**Comment by Testudo-Cultu** [comment]
> Dunno if this was the idea or not but - should heavy crossbow bolts have higher armor penetration than their standard/compound brothers? Because from the video he says that they have higher delivering power - with a heavier bullet this would equate to higher penetration

Since the difference in kinetic energy from the heavy crossbow (using ideal bolts) would be twice of the composite crossbow, yes.

**Comment by estebandellasilva** [comment]
yeah maybe i mixed that up

**Comment by FayFlightee** [comment]
Well if anyone wanna do the heavy bolts stuff for the heavy crossbow as well as damage tweaking, be my guest. I think this discussion is enough.

**Comment by Night-Pryanik** [comment]
Closing as stale, since stalebot can't do it by itself.
